### PR TITLE
Create COMPATIBILITY.md

### PR DESCRIPTION
**Pull request by gxalpha**
Proposal for a compatibility list

### CODE REVIEW

File: COMPATIBILITY.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Compatibility
+
+A list of programs this is currently compatible with.
+Please note that this list is not complete.
+
+|Program|Compatiblity|
+|---|---|
+|Google Chrome|Yes|
+|Zoom|Yes|
```
Comment (johnboiles) [inline COMPATIBILITY.md:9]: Can we add a note about re-codesigning Zoom? Maybe an asterisk that links out to the instructions in one of the issues? Or write up text to go below here.

**Comment by gxalpha** [comment]
Is it issue #4 or #23 ?

**Comment by johnboiles** [comment]
I think there's a bit more about it in #4, but we should maybe simplify the instructions and put them here or in the README. We can do that later tho.

**Comment by danyork** [comment]
The text for Zoom could be something like this:

----

If the virtual camera does not appear in Zoom, a workaround is to quit Zoom, open a Terminal window, and then type:

`codesign --remove-signature /Applications/zoom.us.app/`

When you launch Zoom you should now see OBS available as a camera. Note that unfortunately you may need to run this command after every Zoom upgrade. A better solution is being researched in issue #4 .

----

or something like that.

**Pull request merged**